Source/Library/GameModel.cpp

```diff
@@ -2,16 +2,41 @@
 
 using namespace TicTacToe;
 
-GameModel::GameModel() {}
+GameModel::GameModel() : 
+  m_board{},
```

Comment:
Do not explicitly call the default constructor, it is called automatically.

Reply:
If I don't explicitly call the default constructor, the array variable of Board object will not be initialized to zero.

Reply:
Update ```Board::Board```'s constructor to zero initialize.

Reply:
Do you mean that I add the default constructor in Board class?

Reply:
To clarify I mean that Board's default constructor is required to satisfy the property that ```Board::get_piece(x, y) == Board::Token::NONE``` for all ```0 <= x <= y < 3```. It appears that this isn't the case and hence the fix must be made in ```Board::Board```.